test/parallel/test-readline-promises-interface.js

```diff
@@ -797,6 +797,26 @@ for (let i = 0; i < 12; i++) {
     fi.emit('data', 'asdf\n');
   }
 
+  // Ensure that options.signal.removeEventListener was called
+  {
+    const ac = new AbortController();
+    const signal = ac.signal;
+    const [rli] = getInterface({ terminal });
+    const expectedCallTimes = 1;
```

Comment:
This isn't needed, as it is the default for `common.mustCall()`.

Reply:
nice! thank you! just fixed it!